### DISCUSSION TITLE
User Survey Window Stuck Open

### DISCUSSION THREAD

**Discussion by andyjshepherd** [onlxltd/CommsKit#7] [Bug Reports]
Just updated from an older version. I can't use the software because the Complete User Survey is in the way. No way to close it. Not even a cross button. Survey link Is dead.

Please fix this

**Comment by mdidon** [comment]
@andyjshepherd Thanks for the heads up, we've reopened the survey link.
Could you send a screenshot of the blocked UI.
Thanks

**Comment by andyjshepherd** [comment]
<img width="949" alt="Survey" src="https://github.com/onlxltd/CommsKit/assets/17109097/8d58ff1e-bf41-43e7-a82c-de6fce6d5248">
Please find attached,  Running Ventura, 2020 Macbook Pro

**Comment by mdidon** [comment]
Thanks for that. Looks like an issue with the light mode theme not rendering the buttons as dark. 
In the interim, if you look to the bottom right of this modal window, under the Introduction Ctrl Suite banner, you should just about be able to see two buttons, the left is labelled Don't Show Again, click this.

**Reply by andyjshepherd** [reply]
Got it! It's very faint! Thanks! I'll leave this issue open until it's resolved.

Thanks!

**Comment by mdidon** [comment]
Resolved in v1.8.0
